source/scenes/level/PPLevelSelectScene.cpp

```diff
@@ -27,32 +32,63 @@ void LevelSelectScene::dispose() {
  * to input.
  */
 void LevelSelectScene::activateUI(
-    const std::shared_ptr<cugl::scene2::SceneNode>& scene) {
+    const std::shared_ptr<cugl::scene2::SceneNode>& scene, const char* worldName) {
     std::shared_ptr<scene2::Button> button = std::dynamic_pointer_cast<scene2::Button>(
-        scene);//
+        scene);
+
     if (button != nullptr) {
-                CULog("Activating button %s", button->getName().c_str());
+        //CULog("Activating button %s", button->getName().c_str());
```

Comment:
Remove comments before submitting unless this is necessary to keep for debugging

Reply:
Why? Makes it so much easier when I come back and fix/change things

Reply:
I guess if you may need to change things then leave it. Otherwise it just confuses others that may think this is in flux.